dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -662,44 +524,43 @@ export default defineComponent({
       try {
         this.postEuTaxonomyDataForFinancialsProcessed = false;
         this.messageCount++;
-        const formInputsModelToSend = modifyObjectKeys(
-          JSON.parse(JSON.stringify(this.formInputsModel)) as objectType,
-          "send"
+
+        checkIfAllUploadedReportsAreReferencedInDataModel(
+          this.formInputsModel.data as ObjectType,
+          this.namesOfAllCompanyReportsForTheDataset
         );
+        await (this.$refs.UploadReports.uploadFiles as () => Promise<void>)();
+
+        await this.$nextTick();
```

Comment:
Is this `nextTick` needed here?

Reply:
Working on it

Reply:
Seemingly not necessary, so i removed it.